src/Validation/Validator.php

```diff
@@ -526,6 +582,19 @@ public function allowEmpty($field, $when = true)
      * $validator->notEmpty('email'); // Email cannot be empty
      * $validator->notEmpty('email', $message, 'create'); // Email can be empty on update
      * $validator->notEmpty('email', $message, 'update'); // Email can be empty on create
+     * $validator->notEmpty(['email', 'title'], $message, 'update');
+     * // Email and title can be empty on create but can not on update
+     * $validator->notEmpty(
+     *      [
+     *          'email',
+     *          'title' => [
+     *              'when' => true,
```

Comment:
In reading the rest of this, its the 'on' condition. Any reason to not leave it named 'on'?